_episodes/02-cupy.md

```diff
@@ -236,5 +236,103 @@ The linear convolution is actually performed on the GPU, which is shown by a nic
 
 So this implies a speedup of a factor 104/0.731 = 142. Impressive.
 
+# Using Numba to execute Python code on the GPU
+
+[Numba](http://numba.pydata.org/) is a Python library that "translates Python functions to optimized machine code at runtime using the industry-standard LLVM compiler library". You might want to try it to speed up your code on a CPU. However, Numba also [compiles a subset of Python code into CUDA kernels](https://numba.pydata.org/numba-doc/latest/cuda/overview.html) which is what we will use here. So the idea is that we can do what we are used to, i.e. write Python code and still benefit from the speed that GPUs offer us.
```

Comment:
I would write instead:

However, Numba can also [translate a subset of the Python language into CUDA](https://numba.pydata.org/numba-doc/latest/cuda/overview.html), which is what we will be using here.